### DISCUSSION TITLE
gramedia.com (websites’ own ads)

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1403] [Personal Filter]
![image](https://github.com/realodix/AdBlockID/assets/19517680/0a1e0095-daed-4cd3-be2f-2b4c4c6a60a2)

https://www.gramedia.com/literasi/doa-menyembelih-hewan/#:~:text=5.-,Doa%20Menyembelih%20Sapi,karena%20Allah%20Ta'ala.%E2%80%9D

gambar di sela-sela artikel sangat ganggu banget, hampir menutupi layar monitor
bkn termasuk konten artikel

**Comment by realodix** [comment]
Kerena ini masuk dalam kelompok Websites’ own ads, Saya memindahkannya ke dalam discussions

**Comment by realodix** [comment]
```adblock
gramedia.com##.entry-content > .grame-bukuterbaru
gramedia.com##.entry-content > .grame-pickofthemonth
```
